src/debug/daccess/daccess.cpp

```diff
@@ -40,9 +40,7 @@ HINSTANCE g_thisModule;
 
 extern VOID STDMETHODCALLTYPE TLS_FreeMasterSlotIndex();
 
-#if FEATURE_PAL
-DLLEXPORT // For Win32 PAL LoadLibrary emulation
-#endif
+DLLEXPORT 
```

Comment:
DLLMain needs to be exported for PAL only. This ifdef was here to make that clear.

Reply:
Windows doesn't need DLLMain?

Reply:
It isn't needed on Windows because .def exports it.

Reply:
the .def file

Reply:
DllMain does not need to be exported on Windows. It needs to be exported for PAL only to make DllMain emulation work.

Discussed in https://github.com/dotnet/coreclr/pull/21924#issuecomment-462067477

Reply:
Other than that is the PR ok?